tls/tls_names.go

```diff
@@ -436,6 +436,8 @@ func init() {
 	curveNames[258] = "ffdhe4096"
 	curveNames[259] = "ffdhe6144"
 	curveNames[260] = "ffdhe8192"
+	curveNames[4587] = "secp256r1mlkem768" // draft-kwiatkowski-tls-ecdhe-mlkem
+	curveNames[4588] = "ecdh_x25519mlkem768" // draft-kwiatkowski-tls-ecdhe-mlkem
```

Comment:
Is there a reason this is labeled as `ecdh_x25519mlkem768` and not as `x25519mlkem768`? The latter is how it seems to be defined in `draft-kwiatkowski-tls-ecdhe-mlkem-02`.